app/handlers/status.py

```diff
@@ -23,8 +23,8 @@ class StatusHandler(BaseHandler):
 
     @apiError error_message The error message to display. (Will return unique error message if object tracking has NOT been run on specified project)
     """
-    def post(self):
-        identifier = self.get_body_argument('identifier')
+    def get(self):
+        identifier = self.get_argument('identifier')
```

Comment:
If we are in a get(Self) or post(self) we can use get_argument or get_body_argument safely. If we want to make everything find_argument for consistency, that is fine  (there is no change in behavior).